lib/usd/ui/layerEditor/layerEditorWidget.cpp

```diff
@@ -71,6 +79,8 @@ static void setupButtonWithHIGBitmaps(QPushButton* button, QString baseName)
     QPushButton {
         padding : %1px;
         background-image: url(%2);
+        background-position: center center;
+        background-repeat: no-repeat;
```

Comment:
At 125 the icon used is _100 so there was extra space and the icon was centered and actually repeated. So this centers it (vert/horz) and turns off repeat. I checked all supported scaling factors (100/125/150/200).